### PR TITLE
Portfolio - Artinfo/LV split + new mobile

### PR DESCRIPTION
**Pull request by galdranorn**

### CODE REVIEW

File: _includes/portfolio/carousel.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+    function changeTabPosition(newIndex) {
+      const $activeDotsItem = $carouselDots.find('.portfolio-carousel__dots-item--active');
+      const $newActiveDotsItem = $carouselDots.find($(`[data-dots-index="${newIndex}"]`));
+      const newTabPosition = `-${newIndex * 100}vw`;
+      let $disabledNav = $targetCarousel.find('.portfolio-carousel__nav-item--disabled');
+
+      $disabledNav.removeClass('portfolio-carousel__nav-item--disabled');
+
+      console.log(carouselSize)
```
Comment (galdranorn) [inline _includes/portfolio/carousel.js:69]: console.log removed

File: _sass/new-styles/modules/portfolio/_carousel.scss [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      position: relative;
+
+      &-tab {
+        flex-grow: 1;
+        flex-basis: 0;
+      }
+
+      &--two {
+        width: 200vw !important;
+      }
```
Comment (galdranorn) [inline _sass/new-styles/modules/portfolio/_carousel.scss:26]: not used, removed

File: _includes/portfolio/carousel.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        $disabledNav = null;
+      }
+
+      if ($disabledNav != null) $disabledNav.addClass('portfolio-carousel__nav-item--disabled');
+      $activeDotsItem.removeClass('portfolio-carousel__dots-item--active');
+      $newActiveDotsItem.addClass('portfolio-carousel__dots-item--active');
+      $carouselTabsContainer.animate({ left: newTabPosition }, 600);
+    }
+  }
+})
```
Comment (siaradona) [inline _includes/portfolio/carousel.js:85]: }, { passive: true });

File: _includes/portfolio/carousel.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+      const $activeDotsItem = $carouselDots.find('.portfolio-carousel__dots-item--active');
+      const $newActiveDotsItem = $carouselDots.find($(`[data-dots-index="${newIndex}"]`));
+      const newTabPosition = `-${newIndex * 100}vw`;
+      let $disabledNav = $targetCarousel.find('.portfolio-carousel__nav-item--disabled');
+
+      $disabledNav.removeClass('portfolio-carousel__nav-item--disabled');
+
+      console.log(carouselSize)
+
+      if (newIndex == 0) {
```
Comment (siaradona) [inline _includes/portfolio/carousel.js:71]: `===`

File: _includes/portfolio/carousel.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        $disabledNav = $targetCarousel.find($(`[data-nav="prev"]`));
+      } else if (newIndex == (carouselSize - 1) ) {
```
Comment (siaradona) [inline _includes/portfolio/carousel.js:73]: .

File: _includes/portfolio/lvcloud/video.html [hunk 6]
```diff
@@ -0,0 +1,31 @@
+<div class="portfolio__section portfolio-video">
+  <span class="portfolio__step">Video</span>
+  <h2 class="portfolio__title">Video</h2>
+
+  <div class="portfolio-video__wrapper">
+    <div
+      id="video-cover"
+      class="portfolio-video__cover"
+      style="background-image: url('/assets/images/portfolio/lvcloud/video.png')"
```
Comment (siaradona) [inline _includes/portfolio/lvcloud/video.html:9]: We need to add lightweight background for mobile. Move this to styles and use different background for breakpoints

File: _sass/new-styles/modules/portfolio/_features.scss [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+
+    li {
+      padding: 10px;
+    }
+  }
+}
+
+@media (min-width: 576px) {
+  .portfolio-features__image  {
+    // padding-top: 50px;
```
Comment (siaradona) [inline _sass/new-styles/modules/portfolio/_features.scss:34]: remove

File: _includes/portfolio/carousel.js (hunk 4 above)
Comment (galdranorn) [reply _includes/portfolio/carousel.js:71]: + parseInt

File: _includes/portfolio/carousel.js [hunk 8]
```diff
@@ -51,35 +60,57 @@ window.addEventListener('load', () => {
...
-      const $activeDotsItem = $carouselDots.find('.portfolio-carousel__dots-item--active');
-      const $newActiveDotsItem = $carouselDots.find($(`[data-dots-index="${newIndex}"]`));
-      const newTabPosition = `-${newIndex * 100}vw`;
-      let $disabledNav = $targetCarousel.find('.portfolio-carousel__nav-item--disabled');
+    function checkChangeRequest(newIndex, type = 'default') {
+      const $activeDot = $carouselDots.find('[data-dot-active="true"]');
+      const currentIndex = parseInt($activeDot[0].dataset.dotIndex);
 
-      $disabledNav.removeClass('portfolio-carousel__nav-item--disabled');
+      if (slideChangeGoing === false && (newIndex !== currentIndex || type === 'swipe') {
```
Comment (siaradona) [inline _includes/portfolio/carousel.js:74]: `!slideChangeGoing`

**Pull request merged**